content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -75,34 +77,34 @@ In addition to labels you [attach](#step-one-attach-label-to-the-node), nodes co
 with a standard set of labels. See [Well-Known Labels, Annotations and Taints](/docs/reference/labels-annotations-taints/) for a list of these.
 
 {{< note >}}
-The value of these labels is cloud provider specific and is not guaranteed to be reliable.
+The values of these labels are cloud provider specific and are not guaranteed to be reliable.
 For example, the value of `kubernetes.io/hostname` may be the same as the Node name in some environments
 and a different value in other environments.
 {{< /note >}}
 
 ## Node isolation/restriction
 
 Adding labels to Node objects allows targeting pods to specific nodes or groups of nodes.
-This can be used to ensure specific pods only run on nodes with certain isolation, security, or regulatory properties.
-When using labels for this purpose, choosing label keys that cannot be modified by the kubelet process on the node is strongly recommended.
+This can be used to ensure specific pods  that only run on nodes with certain isolation, security, or regulatory properties.
```

Comment:
Nit


```suggestion
This can be used to ensure that specific pods only run on nodes with certain isolation, security, or regulatory properties.
```